packages/template-retail-react-app/app/pages/checkout/partials/shipping-multi-address.jsx

```diff
@@ -0,0 +1,540 @@
+                                                            {addr.firstName} {addr.lastName} -{' '}
+                                                            {addr.address1},{' '}
+                                                            {formatMessage(
+                                                                {
+                                                                    id: 'shipping_multi_address.format.address_line_2',
+                                                                    defaultMessage:
+                                                                        '{city}, {stateCode} {postalCode}'
+                                                                },
+                                                                {
+                                                                    city: addr.city,
+                                                                    stateCode: addr.stateCode || '',
+                                                                    postalCode: addr.postalCode
+                                                                }
+                                                            )}
+                                                        </option>
+                                                    ))}
+                                                    <option
+                                                        value="add-new"
+                                                        style={{
+                                                            color: '#3182ce',
+                                                            fontWeight: 'bold'
+                                                        }}
+                                                    >
+                                                        + {formatMessage(addNewAddressLabel)}
+                                                    </option>
+                                                </Select>
+                                            </Box>
+
+                                            {/* Price */}
+                                            <Box
+                                                fontWeight="semibold"
+                                                fontSize="md"
+                                                color="gray.900"
+                                                alignSelf="flex-end"
+                                                mt="auto"
+                                                aria-label={formatMessage({
+                                                    id: 'shipping_multi_address.price.label',
+                                                    defaultMessage: 'Product price'
+                                                })}
+                                            >
+                                                {variant.priceAfterItemDiscount && (
```

Comment:
why do we need this condition? The display price component is deisnged to handle at least one price as long as you process the price data correct without having this condition.